src/mock-backend-response.js

```diff
@@ -1,4 +1,4 @@
-const fs = require('fs');
+// const fs = require('fs');
```

Comment:
Shall we remove this obsolete line?

Reply:
Yup, linter should have caught this. I'm used to `lint` running when I do `npm test`, but it was only for `npm run build`.

I've changed `lint` to run on `pretest` now to make sure it's caught in local devleopment.